tutorials/intro/plot_object_annotations.py

```diff
@@ -1,245 +1,234 @@
+#    instantaneous event is desired).
+# 4. Events are stored as an ordinary :class:`NumPy array <numpy.ndarray>`,
+#    whereas :class:`~mne.Annotations` is a :class:`list`-like class defined in
+#    MNE-Python.
 #
-# :class:`~mne.Annotations` supports iterating, indexing and slicing.
-# Iterating over :class:`~mne.Annotations` and indexing with an integer returns
-# a dictionary. While slicing returns a new :class:`~mne.Annotations` instance.
 #
-# See the following examples and usages:
-
-# difference between indexing and slicing a single element
-print(annot[0])  # indexing
-print(annot[:1])  # slicing
-
-###############################################################################
-# How about iterations?
-
-for key, val in annot[0].items():  # iterate on one element which is dictionary
-    print(key, val)
+# What is a STIM channel?
+# ^^^^^^^^^^^^^^^^^^^^^^^
+#
+# Many EEG and MEG systems incorporate one or more recording channels that do
+# not receive signals from a sensor; instead, those channels record voltages
+# (usually short, rectangular DC pulses of fixed magnitudes sent from the
+# experiment-controlling computer) that are time-locked to experimental events,
+# such as the onset of a stimulus or a button-press response by the subject. In
+# other cases, these pulses may not be strictly time-locked to an experimental
+# event, but instead may occur in between trials to indicate the type of
+# stimulus (or experimental condition) that is about to occur on the upcoming
+# trial. In MNE-Python, those channels are referred to as :term:`"STIM"
+# channels <stim channel>`.
+#
+# The DC pulses may be all on one STIM channel (in which case different
+# experimental events or trial types are encoded as different voltage
+# magnitudes), or they may be spread across several channels, in which case the
+# channel(s) on which the pulse(s) occur can be used to encode different events
+# or conditions. Even on systems with multiple STIM channels, there is often
+# one channel that records a weighted sum of the other STIM channels, in such a
+# way that voltage levels on that channel can be unambiguously decoded as
+# particular event types. On older Neuromag systems (such as that used to
+# record the sample data) this was typically the ``STI 014`` channel; on newer
+# systems it is more commonly ``STI101``. You can see the STIM channels in the
+# raw data file here:
+
+raw.copy().pick_types(meg=False, stim=True).plot(start=3, duration=6)
 
 ###############################################################################
-
-for idx, my_annot in enumerate(annot):  # iterate on the Annotations object
-    print('annot #{0}: onset={1}'.format(idx, my_annot['onset']))
-    print('annot #{0}: duration={1}'.format(idx, my_annot['duration']))
-    print('annot #{0}: description={1}'.format(idx, my_annot['description']))
+# Above, you can see that ``STI 014`` (the "sum channel") contains pulses of
```

Comment:
sum channel -> stim channel

Reply:
that was intentional, it's the channel that sums the other stim channels.  Can you think of a clearer way to denote that?

Reply:
hmm ... see my comment below. I learned something today! Although I am not sure if this really should be explained in this tutorial. It seems a Neuromag specific thing.

Reply:
yeah... but the sample data is from a Neuromag system, so some explanation is necessary I think. I've pushed some clarifications; see if you think it's better now.